### ISSUE TITLE
Single field search - Focus styling

### ISSUE THREAD

**Issue by marcellmueller** [bcgov/nr-rec-resources#1128]
#### Description:
Our single field search doesn't have focus styling in admin or the public site. This is a regression from our two field search and should be added for accessibility, though may need a redesign as we have discussed how the current focus styling doesn't look great.
